yoticlient.go

```diff
@@ -208,17 +197,74 @@ func decryptCurrentUserReceipt(receipt *receiptDO, key *rsa.PrivateKey) (result
 	return attributeList, nil
 }
 
+// PerformAmlCheck performs an Anti Money Laundering Check (AML) for a particular user.
+// Returns three boolean values: 'OnPEPList', 'OnWatchList' and 'OnFraudList'.
+func (client *Client) PerformAmlCheck(amlProfile AmlProfile) (AmlResult, error) {
+	return performAmlCheck(amlProfile, doRequest, client.SdkID, client.Key)
+}
+
+func performAmlCheck(amlProfile AmlProfile, requester httpRequester, sdkID string, keyBytes []byte) (result AmlResult, err error) {
+	var key *rsa.PrivateKey
+	var httpMethod = "POST"
```

Comment:
Why don't you make these a series of const values, i.e httpMethodPOST, httpMethodGET, etc.

then when you call createHeaders further down you can just use httpMethodPOST, easy to figure out what it is withouth having to scroll up, plus keeping values like this as const is good practice.